src/mongo/db/pipeline/expression.h

```diff
@@ -443,13 +446,22 @@ class ExpressionFromAccumulator
         return accum.getValue(false);
     }
 
-    bool isAssociativeAndCommutative() const final {
-        // Return false if a single argument is given to avoid a single array argument being treated
-        // as an array instead of as a list of arguments.
+    bool isAssociative() const final {
+        // Return false if a single argument is given to avoid a single array argument being
+        // treated as an array instead of as a list of arguments.
```

Comment:
I assume you didn't mean to change the comment wrapping. Let's keep it wrapped at 100 characters, please. Ditto lines 459-460.